### PR TITLE
Make Run remediation plan accept inputs from Run preupgrade

### PR DESCRIPTION
**Pull request by adamruzicka**
Alternative to #144

### CODE REVIEW

**Comment by stejskalleos** [comment]
@shweta83 can you please verify the fix?

File: app/views/foreman_leapp/job_templates/leapp_remediation.erb [hunk 1]
```diff
@@ -19,6 +19,9 @@ template_inputs:
   required: true
   options: "true\nfalse"
   default: "true"
+foreign_input_sets:
```
Comment (stejskalleos) [inline app/views/foreman_leapp/job_templates/leapp_remediation.erb:22]: @adamruzicka This is the first time I have seen it used. Do we have any user documentation on how & when to use it?

Comment (adamruzicka) [reply app/views/foreman_leapp/job_templates/leapp_remediation.erb:22]: The only other place where it's used is katello. [The old docs](https://theforeman.org/plugins/foreman_remote_execution/1.7/index.html) have something on it - search for "foreign input set"

**Comment by adamruzicka** [comment]
Not sure what's up with the tests. They seem to fail reasonably reliably in GHA on ruby 2.7, yet I'm failing to reproduce locally. The only thing that comes to mind is different load order of seed files, but that's not something I'd like to tackle blindly

**Comment by shweta83** [comment]
@adamruzicka Thanks for the fix. It works well on running remediation and we do not see any error.

**Review by shweta83 — APPROVED** [review]
LGTM. Ack from QE.

**Review by stejskalleos — APPROVED** [review]

**Comment by stejskalleos** [comment]
Thanks @adamruzicka @shweta83

**Pull request merged**